walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -58,8 +56,28 @@ public void setUp() {
     @Test
     public void addJob_NormalInput_success() throws ExecutionException, InterruptedException {
         // Arrange.
+        JobStatus expectedJobStatus = JobStatus.ACTIVE;
         String expectedJobName = "Software Engineer";
-        Job job = new Job(expectedJobName);
+        Location expectedLocation =  new Location("Google", 0, 0);
+        String expectedJobDescription = "Programming using java";
+        JobPayment expectedJobPayment = new JobPayment(0, 5000, PaymentFrequency.MONTHLY);
+        List<String> expectedRequirements = Arrays.asList(
```

Comment:
FYI you can have a utils method in your test (or your enum):
```
static List<String> getLocalizedNames(List<Requirement> reqs, lang){
   ...
}
```

I also recommend another one: (you will have to decide what to do with requirements that no longer exist, probably exclude them instead of throwing.
```
static List<Requirement> getRequirementsFromId(List<String> id){

}
```

Reply:
Fixed with the first version